### ISSUE TITLE
double equal for ==highlighting==?

### ISSUE THREAD

**Issue by michaelbeijer** [adamerose/vscode-markdown-wysiwyg#13]
Hi Adam,

Any chance you could make it so your very cool extension allows users to use some kind of (yellow) text highlighting, like:

Here are ==some very important words==.
Here are `some very important words`.
Here are <mark>some very important words</mark>.

```
Here are ==some very important words==.
Here are `some very important words`.
Here are <mark>some very important words</mark>.
```

?

See: https://stackoverflow.com/questions/25104738/text-highlight-in-markdown

**Comment by michaelbeijer** [comment]
This is what it looks like in Obsidian, which is extremely useful!

![Obsidian](https://github.com/adamerose/vscode-markdown-wysiwyg/assets/16622240/27d5e663-de83-492e-af6e-544a015e2803)

**Comment by adamerose** [comment]
Yeah that would be a good addition. Dropping some more links here for reference

- https://ckeditor.com/docs/ckeditor5/latest/features/highlight.html
- https://ckeditor.com/docs/ckeditor5/latest/features/autoformat.html#creating-custom-autoformatters
- https://www.markdownguide.org/extended-syntax/#highlight
